Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -479,8 +473,8 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
 
     deinit {
         self.notificationCenter.removeObserver(self)
-        self.storeKit1Wrapper?.delegate = nil
-        self.paymentQueueWrapper.delegate = nil
+        self.paymentQueueWrapper.left?.delegate = nil
+        self.paymentQueueWrapper.right?.delegate = nil
```

Comment:
These could potentially use the `storeKit1Wrapper` (and `paymentQueueWrapper` if we add it) extension functions instead of `left` and `right`.

Reply:
Changed